### PR TITLE
Bump vite related dependencies

### PR DESCRIPTION
**Pull request by agualis**
# Description

Bump vite related dependencies.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [x] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: components.d.ts [hunk 1]
```diff
@@ -58,7 +58,6 @@ declare module '@vue/runtime-core' {
     DiscordIcon: typeof import('./src/components/_global/icons/brands/DiscordIcon.vue')['default']
     EmailIcon: typeof import('./src/components/_global/icons/EmailIcon.vue')['default']
     GithubIcon: typeof import('./src/components/_global/icons/brands/GithubIcon.vue')['default']
-    Jazzicon: typeof import('./src/components/_global/JazzIcon/Jazzicon.vue')['default']
```
Comment (garethfuller) [inline components.d.ts:61]: Why was this removed?

**Review by garethfuller — APPROVED** [review]

File: components.d.ts (hunk 1 above)
Comment (agualis) [reply components.d.ts:61]: I investigated a little bit and couldn't find the reason. It is not affecting the DX so I will review it again in case it does.
